staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -305,6 +305,10 @@ func (m *AdmissionMetrics) ObserveWebhookRejection(ctx context.Context, name, st
 	m.webhookRejection.WithContext(ctx).WithLabelValues(name, stepType, operation, string(errorType), strconv.Itoa(rejectionCode)).Inc()
 }
 
+func (m *AdmissionMetrics) WebhookRejectionGathererForTest() *metrics.CounterVec {
```

Comment:
I forgot to add godoc to this, linting will probably complain... can just squash in a comment that this exposes this specific metric for access by unit tests